### DISCUSSION TITLE
AI DAY

### DISCUSSION THREAD

**Discussion by NcuJerryKao** [1101-GS3073-C/Python-Community-GS3073-C#489] [科技新知]
好想有點久了
10月1日 Tesla AI Day 2022
演示Elon Musk團隊的最新成果
https://www.youtube.com/watch?v=ODSJsviD_SU&t=886s&ab_channel=Tesla
以上影片是 AIDAY當天的直播影像，但我看那太久要翻譯又很麻煩，故我看的是以下的解說影片
https://www.youtube.com/watch?v=ztoI4XIoWHs&t=930s&ab_channel=%E7%A7%91%E6%8A%80%E8%B2%A1%E7%9F%A5%E9%81%93
蠻驚訝的 竟會用到不少機械系所學的知識 例如 零力矩點 自由度 ......，果然當初選機械系是對的 機械系是最好的系(By 李勝隆 中大材科教授)

然而 在台灣同月底 也舉辦了一場 AIDAY 我當天有去 不過就跟Tesla AI Day 似乎沒什麼關係 程度也差很多
[【大會手冊】數位產業署AI DAY 2022.pdf](https://github.com/NCU-GS3073-AI-Introduction/ai-introduction-community/files/9958854/AI.DAY.2022.pdf)

**Comment by williamyen0623** [comment]
Tesla 和 執行長Elon Musk都相當有潛力，期待未來能帶給我們許多創新且實用的技術

**Reply by NcuJerryKao** [reply]
有魅力 甚至成為信仰 我蠻喜歡看Elon Musk的TED
想要模仿他的思維模式

**Comment by brian431** [comment]
一直都對人型機器人的發展很有興趣，不知道等我老了之後到底能發展到什麼程度。人型機器人雖然沒辦法在執行特定工作上特別有利，不過在做一些日常雜事上應該是蠻擅長的

**Comment by limitx0** [comment]
機器人能學人走路到這樣自然好厲害
期待以後機器人能做出更多人類的動作

**Comment by skyling0299** [comment]
以前就很喜歡日本的那個機器人，會走路會蹲下會站起來的。雖然當時對AI的定義大概也只有這樣，現在覺得比起像人，功能性更重要

**Comment by Xuewen628** [comment]
機器人會是要結合機器和coding的最好例子 真的
尤其我們對人型機器人的期許越高那他就需要更多關節 更多的可動性
所以機械系事真的很重要沒錯
我愛機械系
